### ISSUE TITLE
Conflict occurs in Electron with the global property 'root'

### ISSUE THREAD

**Issue by abruzzihraig** [grevory/angular-local-storage#288]
In recent release version 0.2.4, the grunt file added some code like below:

```
 root = (typeof root !== 'undefined') ? root : {};
 factory((root.exports || (root.exports = {})), angular);
```

These code would be inserted to file angular-local-storage.js, and then it will cause an error in any web app which runs with Electron.

https://github.com/grevory/angular-local-storage/blob/8a738f918e2407640dfe0f57ad01851b16d33cb2/dist/angular-local-storage.js#L14

It needs to be fixed, thanks.

**Comment by ealves-pt** [comment]
Thanks mate. Do you have any suggestion? I don't have any project that run with Electron so any hint would be helpful.

**Comment by ealves-pt** [comment]
Hmmm... Fun fact is that this shouldn't have been released yet. @grevory did you create the 0.2.4 tag?

**Comment by grevory** [comment]
Yes I did. I saw the PR come in and tagged it for release. I didn't realize
at the time that you were on the ball.


Greg

**Comment by grevory** [comment]
Don't forget to update the version on new commits. I see a few others in there. Thanks!

**Comment by ealves-pt** [comment]
This should be fixed with PR #291. I will wait for some feedback to see if it's possible to close this issue.
